packages/pluggableWidgets/datagrid-web/src/components/Table.tsx

```diff
@@ -313,14 +314,17 @@ export function Table<T>(props: TableProps<T>): ReactElement {
                     {(isSortingOrFiltering && props.paging ? rowsPagination : rows).map((row, rowIndex) => {
                         prepareRow(row);
                         return (
-                            <Fragment key={`row_${rowIndex}`}>
+                            <td
```

Comment:
Are the cells rendered inside this td's as well?

Reply:
Its a pseudo-td, it doesnt render because of `display: contents;` it actually just show the content of the td. You can apply styles in the tr that targets the tds, but nothing around tr itself.